lms/static/scripts/frontend_apps/components/LaunchErrorDialog.js

```diff
@@ -161,21 +160,52 @@ export default function LaunchErrorDialog({
           </p>
         </ErrorModal>
       );
+    case 'd2l_group_set_not_found':
```

Comment:
For these everything is in place to just create a new branch and do the full dialog.